src/scripts/main.js

```diff
@@ -1 +1,76 @@
 'use strict';
+
+const createNotification = (messege, classList) => {
+  const element = document.createElement('div');
+
+  element.textContent = messege;
+  element.classList.add('success', classList);
+  document.body.append(element);
+};
+
+const firstPromise = () => {
+  const resolver = (resolve, reject) => {
+    document.addEventListener('click', () => {
+      resolve();
+    });
+
+    setTimeout(() => {
+      reject();
+    }, 3000);
+  };
+
+  return new Promise(resolver);
+};
```

Comment:
```suggestion
const firstPromise = new Promise((resolve, reject) => {
  document.addEventListener('click', () => {
    resolve();
  });

  setTimeout(() => {
    reject();
  }, 3000);
});
```
Зачем тут дополнительная обертка из функции?